src/components/scenes/FormScene.tsx

```diff
@@ -9,6 +9,7 @@ import { SceneHeader } from '../themed/SceneHeader'
 
 interface OwnProps {
   headerText: string
+  headerIcon?: React.ReactNode
```

Comment:
Same comment as the other file, headerTooltipIcon. Make this more distinct than just headerIcon to avoid confusion with static left-justified icons.

Reply:
`headerTertiary`